lib/internal/timers.js

```diff
@@ -137,7 +138,7 @@ let timerListId = NumberMIN_SAFE_INTEGER;
 const kRefed = Symbol('refed');
 
 let nextExpiry = Infinity;
-let refCount = 0;
+timeoutInfo[0] = 0;
```

Comment:
Can you add a comment and explain what `timeoutInfo[0]` represents?

Reply:
done!